libs/island-ui/core/src/lib/Input/Input.mixins.ts

```diff
@@ -200,6 +200,9 @@ export const optionDescriptionSizes = {
 export const inputPlaceholder = {
   color: theme.color.dark300,
   fontWeight: theme.typography.light,
+
+  // Firefox applies 0.54 opacity to placeholder text. This disables that.
+  opacity: 1,
 }
 
 export const placeholder = {
```

Comment:
I wonder if it'd also be beneficial to add the same styling on the placeholder class since it looks like it's being used for the Select component. Could you check if that is also having the firefox issue regarding opacity not being 1?

Reply:
Dropdown placeholders do not have this issue on Firefox. The LÖKE input at the top is a dropdown..

<img width="424" alt="Justice League Silence of the lambs Lota 28 Screenshot Feb 23" src="https://github.com/island-is/island.is/assets/3789875/83c4c305-1242-4316-b734-e45f8862fbe2">